src/main/java/org/jitsi/utils/dsi/DominantSpeakerIdentification.java

```diff
@@ -431,6 +482,96 @@ private synchronized Speaker getOrCreateSpeaker(T id)
         return speaker;
     }
 
+    private synchronized void updateLoudestList(Speaker speaker, int level, long now)
+    {
+        long oldestValid = now - energyExpireTimeMs;
+
+        /* exponential smoothing */
+        speaker.energyScore = (energyAlphaPct * level + (100 - energyAlphaPct) * speaker.energyScore) / 100;
```

Comment:
Should the alpha factor be a declared constant?

Reply:
I think `energyAlphaPct` can't be a constant, because it's configurable?

I don't know if this is important, but integer division takes the floor of the result (so 99/100 is 0). You can achieve rounding by adding half the number being divided.

Reply:
yes it's configurable.

agreed, rounding to nearest now.